packs/equipment/hodag-leather.json

```diff
@@ -0,0 +1,75 @@
+{
+    "_id": "EnTfjcVhcdgd1sLW",
+    "img": "systems/pf2e/icons/default-icons/armor.svg",
+    "name": "Hodag Leather",
+    "system": {
+        "acBonus": 2,
+        "baseItem": "studded-leather-armor",
+        "bulk": {
+            "value": 1
+        },
+        "category": "light",
+        "checkPenalty": -1,
+        "containerId": null,
+        "description": {
+            "value": "<p>Dyed green and white and studded with the natural spines of a hodag, this +1 deathless (Grand Bazaar 10) studded leather armor is an imposing sight, especially when its wearer is barreling towards you.</p>\n<p><strong>Activate—Hodag Toss <span class=\"action-glyph\">2</span></strong> (concentrate, manipulate) <strong>Effect</strong> You Stride up to your Speed and make a Strike at the end of your movement. If you succeed at your Strike, attempt an Athletics check against the creature's Fortitude DC. On a success, the creature is thrown 10 feet in a straight line in the direction of your choice and lands @UUID[Compendium.pf2e.conditionitems.Item.Prone]. If the creature is knocked into a solid object, it takes 1d6 bludgeoning damage before landing prone. You can instead toss the creature straight up into the air. The creature lands in the same square it occupied, and takes @Damage[1d6[bludgeoning]] damage as it lands prone. You can't use this ability again for [[/br 1d4 #Recharge this ability]]{1d4 rounds}.</p>\n<hr />\n<p><strong>Craft Requirements</strong> The initial raw materials must include the hide of a hodag.</p>"
```

Comment:
Consider linking to deathless rune rather than listing source and page number.

Reply:
We don't generally link in these cases, but we should emphasize and remove the page reference.